content/blog/2019/istio1.1_perf/index.md

```diff
@@ -0,0 +1,57 @@
+
+Now, new enhancements in Istio 1.1 deliver scale-up with improved application performance and service management efficiency.
+Simulations using our sample commercial airline reservation application show the following improvements, compared to Istio 1.0.
+
+###### Application performance ######
+
+* up to 30% reduction in application average latency
+* up to 40% faster service startup times in a large mesh
+
+###### Service management efficiency ######
+
+* up to 90% reduction in Pilot CPU usage in a large mesh
+* up to 50% reduction in Pilot memory usage in a large mesh
+
+With Istio 1.1, organizations can be more confident in their ability to scale applications with consistency and control -- even in hyper-scale cloud environments.
+Congratulations to the Istio experts around the world who contributed to this release. We could not be more pleased with these results.
+
+##### Istio 1.1 performance enhancements
+
+As members of the Istio Performance and Scalability workgroup, we have done extensive performance evaluations. We introduced many performance design features for Istio 1.1, in collaboration with other Istio contributors.
+Some of the most visible performance enhancements in 1.1 include:
+
+* Significant reduction in default collection of Envoy-generated statistics
+* Added load-shedding functionality to Mixer workloads
+* Improved the protocol between Envoy and Mixer
+* Namespace isolation, to reduce operational overhead
+* Configurable concurrent worker threads, which can improve overall throughput
+* Configurable filters that limit telemetry data
+* Removal of synchronization bottlenecks
+
+Regression Patrol drives continuous improvement in Istio performance and quality. Behind the scenes, the Regression Patrol Framework helps Istio developers to identify and fix code issues. Daily builds are checked using a customer centric benchmark, [BluePerf](https://github.com/blueperf/). The results are published to the [Istio community web portal](https://ibmcloud-perf.istio.io/regpatrol/). Various application configurations are evaluated to help provide insights on Istio component performance.
+
+##### Summary
+
+Istio 1.1 was designed for performance and scalability. The Istio Performance and Scalability workgroup measured significant performance improvements over 1.0.
+Istio 1.1 introduces new features and optimizations to help harden the service mesh for enterprise microservice workloads. The [Istio 1.1 Performance and Tuning Guide](../../../help/ops/perf/) documents performance simulations, provides sizing and capacity planning guidance, and includes best practices for tuning custom use cases.
```

Comment:
Here and elsewhere, please replace all .. paths with absolute paths starting with a /. In this case /help/ops/perf

Reply:
done